core/src/components/picker-column/picker-column.tsx

```diff
@@ -325,8 +338,8 @@ export class PickerColumn implements ComponentInterface {
           }
         }
 
-        activeEl = activeElement;
-        this.setPickerItemActiveState(activeElement, true);
+        activeEl = newActiveElement;
```

Comment:
The name change isn't needed to land this PR, but I couldn't immediately tell the different between `activeEl` and `activeElement`